src/Service.GraphQLBuilder/Queries/StandardQueryInputs.cs

```diff
@@ -169,7 +169,6 @@ public static InputObjectTypeDefinitionNode StringInputType() =>
                     new(null, new NameNode("startsWith"), new StringValueNode("Starts With"), new StringType().ToTypeNode(), null, new List<DirectiveNode>()),
                     new(null, new NameNode("endsWith"), new StringValueNode("Ends With"), new StringType().ToTypeNode(), null, new List<DirectiveNode>()),
                     new(null, new NameNode("neq"), new StringValueNode("Not Equals"), new StringType().ToTypeNode(), null, new List<DirectiveNode>()),
-                    new(null, new NameNode("caseInsensitive"), new StringValueNode("Case Insensitive"), new BooleanType().ToTypeNode(), new BooleanValueNode(false), new List<DirectiveNode>()),
```

Comment:
nit: fix title typo, should be:  `caseInsensitive`

Reply:
Fixed